package.json

```diff
@@ -21,6 +21,7 @@
     "build": "",
     "before-test": "node tests/fixtures/link.js",
     "test": "npm run before-test && mocha \"tests/*.js\"",
+    "test-no-int": "npm run before-test && mocha tests/no-internal.js",
```

Comment:
remove this before merging; can do the same thing with test and just passing in grep flag

Reply:
Oops this was not meant to be committed, removed